types/lib/model.d.ts

```diff
@@ -1825,25 +1828,43 @@ export abstract class Model<TModelAttributes extends {} = any, TCreationAttribut
   public static findByPk<M extends Model>(
     this: ModelStatic<M>,
     identifier: Identifier,
-    options: Omit<NonNullFindOptions<M['_attributes']>, 'where'>
+    options: Omit<NonNullFindOptions<M['_attributes']>, 'where'> & { raw?: false }
   ): Promise<M>;
+  public static findByPk<M extends Model, R extends any = M['_attributes']>(
```

Comment:
Continuing conversation from https://github.com/sequelize/sequelize/pull/13219#discussion_r642108698

@AllAwesome497 this is the PR for raw addition.

See attached issue, but the presence of `raw: true` does more than expose attributes, but allows overriding that type for whatever the response should be.

Reply:
yea, my comment on the other PR should also allow that but without the overloading, if TS is smart enough to infer raw's type as its value, not the type of the object (eg, as having the property `raw` as true instead of `raw?: boolean`)